### PR TITLE
QUEST #1218-1235

### PR DESCRIPTION
**Pull request by vtange**
End of Modestas story. Rough translation, editing can come later.
Will be working on the remaining lines mentioned in Issue #922 in another branch

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -1214,24 +1214,24 @@ QUEST_20150317_001214	Modestas knew the these gambles were more than just about
...
+QUEST_20150317_001221	The officer and solders had sturdy rope, so they were able to transport the rock without worry of it falling off and unload it at Klaipeda. They decided to accompany Modestas all the way there. {nl} Modestas was very grateful for the help he got from the soldiers. {nl} Along with the soldiers, Modestas again ran into none other than the Alchemist he lost earlier against.
+QUEST_20150317_001222	To Modestas, a promise was a promise, so he immediately handed over the gold the soldiers returned to him to the Alchemist. {nl} The Alchemist, surprised by how Modestas could do this with no complaints or dissatisfaction, took the gold and conversed with him.
+QUEST_20150317_001223	Not wanting to be left out of the conversation between Modestas and the Alchemist, the soldiers couldn't continue working, joined in, and learned about the Alchemist's past with Modestas. {nl} The Alchemist apologized to Modestas for turning his lead into gold, and promised to gather all the support he could for Modestas at Klaipeda. He then left. {nl}
```
Comment (Soukyuu) [inline QUEST.tsv:1222]: Wait, I thought the old lady was the Alchemist? Suddenly she's male?

File: QUEST.tsv [hunk 2]
```diff
@@ -1214,24 +1214,24 @@ QUEST_20150317_001214	Modestas knew the these gambles were more than just about
...
+QUEST_20150317_001224	Soon after, Modestas and the soldiers reached Klaipeda. They set the stone down at the large open space at the center of town. {nl}
+QUEST_20150317_001225	{memo X}When the boulder was set in its proper place with the help of the soldiers, the Alchemist who had left returned with another person. The Alchemist had brought with him a famous Dievdirbys sculptor.{nl}To the famous Dievdirbys the Alchemist suggested.{nl}[Won't you sculpt a statue of a Goddess using this boulder, on this central site of Klaipeda? If this boulder is changed into a statue of a Goddess, then nobody in Klaipeda will be able to deny its significance. I think that will fulfill your promise to the old lady.]{nl}
```
Comment (Soukyuu) [inline QUEST.tsv:1224]: We're using single quotes for dialogue now.

File: QUEST.tsv [hunk 3]
```diff
@@ -1214,24 +1214,24 @@ QUEST_20150317_001214	Modestas knew the these gambles were more than just about
...
+QUEST_20150317_001232	And so, some people believed Modestas, having undertaken the Goddesses' intervention, was an honest man who did not gamble. In reality, Modestas didn't stop making gambles until his death. {nl}
+QUEST_20150317_001233	{memo X}Since that time, whenever Modestas took on a wager, he always betted the same thing. The answer to the question of who the Statue of Klaipeda honored.{nl}To uncover the name of the Goddess, numerous people challenged Modestas to a bet.{nl} {nl} But no matter the impossible odds, Modestas never lost a bet. Therefore nobody was able to win the answer from Modestas for years and time flew by.{nl}
+QUEST_20150317_001234	Though Modestas died an undefeated, invincible gambler, by then he had as mentioned never bet anything other than the Goddess' identity, living purely off the profits gained from such bets through the end of his life. {nl}
```
Comment (Soukyuu) [inline QUEST.tsv:1233]: Rather confusingly formulated.

File: QUEST.tsv (hunk 1 above)
Comment (vtange) [inline QUEST.tsv:1222]: the Old lady and Alchemist are two separate characters. 
Major Characters in this story:
- Modestas
- Old Lady (3rd/final loss Modestas)
- Sigfried, Modesta's friend (first win Modestas)
- A bunch of soldiers and their officer (first loss Modestas)
- A man with a injured horse and a nice wagon (2nd win Modestas)
- An Alchemist who tricks Modestas with lead into gold (2nd loss Modestas)
- A mountain (3rd win Modestas)
- A Dievdirby who carves the statue

File: QUEST.tsv (hunk 2 above)
Comment (vtange) [inline QUEST.tsv:1224]: Oh I see, I had used double quotes for Aiste's story above as well. I can do a mass replace of them in another branch.

File: QUEST.tsv (hunk 3 above)
Comment (vtange) [inline QUEST.tsv:1233]: Agree, the story seemed to fall in place really quickly, probably too quickly, and a lot of small details were added that only thicken the language. But this is what I got from scratching out every bit of the original Korean text.

File: QUEST.tsv (hunk 2 above)
Comment (Soukyuu) [inline QUEST.tsv:1224]: From what I heard, regular double quotes break ingame display (but no the unicode ones, or something). Just FYI

Comment (Soukyuu) [inline QUEST.tsv:1224]: I just realized it's a {memo X} line, it's not used anywhere.

File: QUEST.tsv (hunk 1 above)
Comment (Soukyuu) [inline QUEST.tsv:1222]: I see, nvm then.

**Comment by Soukyuu** [comment]
:+1:

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**